Remotion/Web/Core/UI/SmartPageEvents.cs

```diff
@@ -34,8 +34,6 @@ public enum SmartPageEvents
     OnAbort,
     /// <summary> Raised when the user scrolls the page. Signature: <c>void Function ()</c> </summary>
     OnScroll,
-    /// <summary> Raised when the user resizes the page. Signature: <c>void Function ()</c> </summary>
-    OnResize,
```

Comment:
Considering there are (very) few usages, but not none, we should keep the value and mark it as Obsolete(error=true). Message: `"SmartPage.OnResize event is no longer supported. Use CSS (recommendation) or implement a resize handler inside application code. (Version 4.0.0)"`